From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 23 Feb 2015 03:29:32 +0000
Subject: sock: Fix ABI change in 3.2.67

'Revert "tcp: Apply device TSO segment limit earlier"' in 3.2.67
removed sock::sk_gso_max_segs.  Add it back.

--- a/include/net/sock.h
+++ b/include/net/sock.h
@@ -310,6 +310,7 @@ struct sock {
 	int			sk_route_nocaps;
 	int			sk_gso_type;
 	unsigned int		sk_gso_max_size;
+	u16			sk_gso_max_segs;	/* now unused */
 	int			sk_rcvlowat;
 	unsigned long	        sk_lingertime;
 	struct sk_buff_head	sk_error_queue;
